XenModel/Messages.resx

```diff
@@ -13567,4 +13565,16 @@ You will need to navigate to the Console on each of the selected VMs to complete
   <data name="YOU_ARE_HERE" xml:space="preserve">
     <value>You are here</value>
   </data>
+  <data name="CHOOSE_SR_PROVISIONING_PAGE_TITLE" xml:space="preserve">

```

Comment:
It looks like the Messages.resx file has become unsorted. We should sort it alphabetically.